packages/stores/src/mockStore.ts

```diff
@@ -629,7 +631,7 @@ const mock = (): TStores & { is_mock: boolean } => {
             removeAccountSwitcherListener: jest.fn(),
             setAccountSwitcherListener: jest.fn(),
         },
-        chart_barrier_store: {},
+        chart_barrier_store: {} as unknown as TCoreStores['chart_barrier_store'],
```

Comment:
`chart_barrier_store: {} as TCoreStores['chart_barrier_store']`

`chart_barrier_store` is just a record type. I think it should work without converting the type to unknown.

<img width="379" alt="Screenshot 2023-11-08 at 1 18 34 PM" src="https://github.com/kate-deriv/deriv-app/assets/56330681/40bbea6c-c278-4775-a59a-6c2fc3a251d6">

Reply:
@balakrishna-deriv unfortunately, it doesn't work, i tried